<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperatorSliders</class>
 <widget class="QWidget" name="OperatorSliders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="operatorNumberLabel">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>number</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LabeledVSlider" name="arSlider" native="true">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LabeledVSlider" name="drSlider" native="true">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LabeledVSlider" name="srSlider" native="true">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LabeledVSlider" name="rrSlider" native="true">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LabeledVSlider" name="slSlider" native="true">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LabeledVSlider" name="tlSlider" native="true">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LabeledVSlider" name="ksSlider" native="true">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LabeledVSlider" name="mlSlider" native="true">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LabeledVSlider" name="dtSlider" native="true">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LabeledVSlider" name="amSlider" native="true">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabeledVSlider</class>
   <extends>QWidget</extends>
   <header>widgets/labeled_vslider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
